backend/integraflow/messaging/tasks.py

```diff
@@ -45,17 +49,24 @@ def send_invite(invite_id: str) -> None:
     ).get(
         id=invite_id
     )
+
+    if invite is None:
+        return
+
+    link = f"/invite/{invite.pk}/accept"
     message = EmailMessage(
         campaign_key=campaign_key,
         subject=(
-            f"{invite.created_by.first_name} invited you to join "
-            f"{invite.organization.name} on Integraflow"
+            f"{invite.created_by.email or invite.created_by.first_name} "
```

Comment:
```suggestion
            f"{invite.created_by.first_name or invite.created_by.email} "
```
@jdamilola shouldn't `invite.created_by.first_name` come first?